labgrid/target.py

```diff
@@ -97,15 +97,15 @@ def await_resources(self, resources, timeout=None, avail=True):
 
         self.update_resources()
 
-    def get_resource(self, cls, *, name=None, await=True):
+    def get_resource(self, cls, *, name=None, wait_for=True):
         """
         Helper function to get a resource of the target.
         Returns the first valid resource found, otherwise None.
 
         Arguments:
         cls -- resource-class to return as a resource
         name -- optional name to use as a filter
-        await -- wait for the resource to become available (default True)
+        wait_for -- wait for the resource to become available (default True)
```

Comment:
I'd prefer wait_avail instead of wait_for. wait_for suggests that you need to pass the thing you want to wait for.